lib/mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.cpp

```diff
@@ -348,11 +352,23 @@ bool ProxyRenderDelegate::enableUpdateForSelection() const { return true; }
 #endif
 
 //! \brief  Always requires update since changes are tracked by Hydraw change tracker and it will
-//! guarantee minimal update
+//! guarantee minimal update; only exception is if rendering through Maya-to-Hydra
 bool ProxyRenderDelegate::requiresUpdate(
     const MSubSceneContainer& container,
     const MFrameContext&      frameContext) const
 {
+#if defined(BUILD_HDMAYA)
+    std::cout << "ProxyRenderDelegate::update - BUILD_HDMAYA - checking if IsMtohRenderOverride"
+              << std::endl;
+
+    // If the current viewport renderer is an mtoh one, skip this update, as
+    // mtoh already has special handling for proxy shapes, and we don't want to
+    // build out a render index we don't need
+    if (IsMtohRenderOverride(frameContext)) {
+        return false;
+    }
+    std::cout << "ProxyRenderDelegate::requiresSpdate - was not mtoh" << std::endl;
```

Comment:
Could you remove the prints at Line 361 and Line 370 which will get shown for every refresh? Or at least protect it with a `TF_DEBUG` option.

Reply:
Or change to print some message when it is using mtoh render override (with TF_DEBUG condition).

Reply:
Oops - yes, absolutely - accidentally had left some debug prints in.

Removed in https://github.com/Autodesk/maya-usd/pull/1195/commits/03890ff59a4c347bb5107445bb710e8f6c4eb01d